allure-app-under-test/src/main/java/com/capgemini/ntc/selenium/pages/projectY/HorizontalSliderPage.java

```diff
@@ -0,0 +1,231 @@
+					case HorizontalSliderPage.KEYBOARD:
+						performKeypress(Keys.ARROW_RIGHT);
+						break;
+					case HorizontalSliderPage.MOUSE:
+						performMouseMove(MOUSE_MOVE_RIGHT);
+						break;
+					default:
+						BFLogger.logDebug(message + method);
+						break;
+				}
+			} else if (position.compareTo(horizontalSlider.getCurrentSliderValue()) < 0) {
+				switch (method) {
+					case HorizontalSliderPage.KEYBOARD:
+						performKeypress(Keys.ARROW_LEFT);
+						break;
+					case HorizontalSliderPage.MOUSE:
+						performMouseMove(MOUSE_MOVE_LEFT);
+						break;
+					default:
+						BFLogger.logDebug(message + method);
+						break;
+				}
+			}
+			counter++;
+		}
+	}
+	
+	/**
+	 * Verifies if given value can be a proper position of horizontal slider and correct it if necessary.
+	 * <p>
+	 * If given value is less then the minimum possible value then returned value is the minimum.<br>
+	 * If given value is more than the maximum possible value then returned value is the maximum.<br>
+	 * </p>
+	 * <p>
+	 * If given value is not a multiply of step defined for horizontal slider, then it is rounded to
+	 * nearest multiply of step.
+	 * <ul>
+	 * <li>If difference between step and scaled value is more than or equal to half of step value, then
+	 * position value is rounded up, e.g. step=0.5, position=1.25 is rounded to 1.5.
+	 * <li>Otherwise position value is rounded down, e.g. step=0.5, position=1.13 is rounded to 1.
+	 * </ul>
+	 * </p>
+	 * 
+	 * @param position
+	 *            Given value of slider's position.
+	 * @return BigDecimal representing correct value of slider's position.
+	 * @see HorizontalSliderElement
+	 * @see BigDecimal
+	 */
+	public BigDecimal verifyAndCorrectPositionValue(BigDecimal position) {
+		if (position.compareTo(getStartPosition()) == -1) {
+			BFLogger.logInfo("Position value: " + position + " will be set to minimum value: " + getStartPosition());
+			position = getStartPosition();
+		} else if (position.compareTo(getEndPosition()) == 1) {
+			BFLogger.logInfo("Position value: " + position + " will be set to maximum value: " + getEndPosition());
+			position = getEndPosition();
+		}
+		BigDecimal step = horizontalSlider.getStep();
+		BigDecimal rest = position.remainder(step);
+		if (rest.compareTo(BigDecimal.ZERO) != 0) {
+			BigDecimal oldPosition = position;
+			BigDecimal sub = step.subtract(rest);
+			int result = sub.compareTo(rest);
+			switch (result) {
+				case -1:
+				case 0:
+					position = position.add(sub, new MathContext(step.scale() + 1));
+					break;
+				case 1:
+					position = position.subtract(rest, new MathContext(step.scale() + 1));
+					break;
+				default:
+					break;
+			}
+			if (position.scale() > 0)
+				position = position.stripTrailingZeros();
+			BFLogger.logInfo("Position value: " + oldPosition + " must be a multiply of " + step + ". It will be rounded to: " + position);
+		}
+		return position;
+	}
+	
+	private void performLeftMouseClickOnWebElement() {
```

Comment:
You could use    DriverExtention.performLeftMouseClickOnWebElement(By selector)  from  PR   https://github.com/devonfw/devonfw-testing/pull/75/files#

Reply:
Indeed, done this way.